multicluster/charts/linkerd-multicluster/templates/gateway-policy.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     matchLabels:
       app: {{.Values.gateway.name}}
   port: linkerd-proxy
-  proxyProtocol: HTTP/1
```

Comment:
This is probably a good change either way, but as noted in #9574, this may not be sufficient to address the underlying problem described in the issue.